OpenRA.Mods.Common/UpdateRules/Rules/20210321/RemoveRapairableNear.cs

```diff
@@ -0,0 +1,37 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2022 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RemoveRepairableNear : UpdateRule
+	{
+		public override string Name => "Merge RepairableNear into Repairable.";
+
+		public override string Description => "RepairableNear was obsoleted. Use Repairable with CloseEnough instead.";
+
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			foreach (var repairable in actorNode.ChildrenMatching("RepairableNear"))
+			{
+				repairable.RenameKey("Repairable");
+
+				var close = repairable.ChildrenContaining("CloseEnough");
+				if (!close.Any())
+					repairable.AddNode("CloseEnough", "4c0");
```

Comment:
This will add a node to negated traits:
![image](https://user-images.githubusercontent.com/7704140/185255006-b4afcc19-cd04-4c5b-b7eb-bdcf2333c46c.png)
(Renaming `-RepairableNear` to `-Repairable` works correctly.)

Reply:
Added a IsRemoval() check